python/graphscope/client/session.py

```diff
@@ -894,14 +894,19 @@ def group_property_types(props):
         handle_json_string = json.dumps(handle)
         return base64.b64encode(handle_json_string.encode("utf-8")).decode("utf-8")
 
-    def gremlin(self, graph, engine_params=None):
+    @set_defaults(gs_config)
+    def gremlin(self, graph, engine_params=None, timeout_seconds=60):
         """Get a interactive engine handler to execute gremlin queries.
 
+        Note that this method will be executed implicitly when a property graph created.
+
         Args:
             graph (:class:`Graph`): Use the graph to create interactive instance.
             engine_params (dict, optional): Configure startup parameters of interactive engine.
+                You can also configure this param by `graphscope.set_option(engine_params={})`.
                 See a list of configurable keys in
                 `interactive_engine/deploy/docker/dockerfile/executor.vineyard.properties`
+            timeout_seconds (int, optional): For waiting gremlin service ready. Defaults to 60.
```

Comment:
Since a gremlin is automatically called when graph is loaded, how can user configure the timeout parameter?

Reply:
graphscope.set_option(timeout_seconds=100)

Reply:
If this is the case, maybe the actual timeout value is 600 by default. See in:
https://github.com/alibaba/GraphScope/blob/10f36b1dee462bf3efe9439a354e4f9e6d9f6da1/python/graphscope/config.py#L80